README.md

```diff
@@ -1,634 +1,1855 @@
-   
-2. **Lender claims a finished loan** - The lender cannot claim a loan unless it is either fully paid or it is expired. If the lender set the time to be earlier than it actually is, the loan is still active and the lender cannot claim it. If the lender set the time to be later than it actually is the transaction would not (yet) be valid, so the lender would have to wait longer to claim what they are owed. These two together result in **the lender being incentivized to set `invalid-before` to be as close as the current time as possible.**
-
-3. **Borrower makes a payment** - The borrower cannot make payments once a loan has expired. This is enforced by the borrower having to use `invalid-hereafter` to specify the current time. Since the borrower wants as much time as possible to pay off the loan, but *must* do so prior to expiration slot, **they are incentivized to set `invalid-hereafter` as close the expiration slot as possible.**
-
-#### Interest
-Unlike other lending/borrowering protocols, Cardano-Loans does not use an algorithm to determine the interest rates. Instead, rates are one of the explicitly negotiated terms between borrowers and lenders.
-
-**Cardano-Loans v1 utilizes fixed, non-compounding interest rates.** Validity intervals cannot tell the current time during payments, they can only tell whether a certain time has passed or not. Compounding requires the state of the loan to be periodically updated, which is possible by breaking the loan up into "epochs", using `invalid-hereafter` to check whether the current loan-epoch has passed. This is similar to the expiry check that occurs when the borrower makes a repayment, though it adds complexity and logic to the design. For a more detailed explanation, see the [Future Directions](#future-directions-and-considerations) section.
-
-Since Cardano-Loans v1 is non-compounding, the total amount owed for all loans is always:
-```
-amountOwed = principle * (1 + interestRate)
+This section is the low-level specification for the protocol. If you are only interested in the
+high-level aspects, feel free to skip to the next [section](#benchmarks-and-fee-estimations-ymmv).
+
+### The Borrower's Address
+
+Borrowers each create a unique "loan" address - this is where all negotiations take place *and*
+where all collateral is kept until the loan is either repaid or expired. As is common in
+*distributed dApps*, all such loan addresses use the same validator script for the payment
+credential, and a unique, user-defined staking key or script for the staking credential.
+Owner-related actions are delegated to the staking credential *by* the validator script, so the user
+maintains full control of all assets at the address. The borrower's credit history is
```

Comment:
Full control?  Presumably they can't spend their tokens.

Reply:
Full *custody* is probably better. They can't spend locked collateral, but they are the only entity that possibly could.